app/main.py

```diff
@@ -1,8 +1,17 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = [Person(one["name"], one["age"]) for one in people]
```

Comment:
```suggestion
    persons_list = [Person(person["name"], person["age"]) for person in people]
```
one - is bad variable name